src/app/Dashboard/Dashboard.tsx

```diff
@@ -1,12 +1,10 @@
 import * as React from 'react';
 import { PageSection, Title } from '@patternfly/react-core';
 
-const Dashboard: React.FunctionComponent<any> = (props) => {
-  return (
+const Dashboard: React.FunctionComponent<{}> = () => (
```

Comment:
This is the first time I've seen the `<{}>` syntax, do you know what the deal is with that? Is it just another way to write `any`?

Reply:
I believe this just signifies that the component doesn't take or require any props at all while supplying a type of `any` would allow any type of props to be passed.